mautic/mautic
# [UX] Change "Stop engaging after a conversion?" to a positive statement

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
When creating a focus item, you have this option to decide whether the item should try again with the same user, at another time, if the user already has converted the first time.
This PR changes the action to a positive question.

Before: `Stop engaging after a conversion?`
After: `Engage again after a conversion?`



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Channels > Focus items > New > Builder
3. Select the type "Collect data" and see the new label

![image](https://github.com/mautic/mautic/assets/116097999/ff7afc3a-cf84-4558-96d5-835d080fe5c2)
